odoo/addons/base/models/res_lang.py

```diff
@@ -249,6 +249,25 @@ def _lang_code_to_urlcode(self, code):
                 return urlc
         return self._lang_get(code).url_code
 
+    @api.model
+    def _get_enabled_lang_code(self):
+        """ Get the list of code enabled (aka installed and default) """
+        langs = self.get_enabled()
+        return [lang[0] for lang in langs]
+
+    @api.model
+    @tools.ormcache()
+    def get_enabled(self):
+        """
+        Return the installed languages and the default one (aka iso_code is only 2 letters)
+        as a list of (code, name) sorted by the active state and the name (default order of model)
+        """
+        langs = self.with_context(active_test=False).search([])
+        lang_4iso_desactivated = langs.filtered(lambda lang: not lang.active and len(lang.iso_code) > 2)
+        langs_enabled = langs - lang_4iso_desactivated
+        name_get = lambda lang: lang.name if lang.active else "%s %s"  % (lang.name, _("(not installed)"))
```

Comment:
Could call name_get instead (and rebuild a final list using zip, much nice python)